FlightsWebsite/src/js/weather.js

```diff
@@ -1,15 +1,6 @@
-const URL = 'https://flights-service-buki55n7ba-lz.a.run.app/weather';
-// const URL = 'http://localhost:8080/weather';
-
 function selectedWeather() {
     const option = document.getElementById("weather");
     switch (option.selectedIndex) {
-        case 1:
```

Comment:
I have a feeling you have merged incorrectly this file .. as cases 1 and 2 are missing together with function for displaying appropriate graphs

Reply:
Merged manually and updated PR